3-4-1/example_config.R

```diff
@@ -0,0 +1,21 @@
+# configurations for type 4 (Nomis) data
+
+# link for data - see the readme
+# Nomis link generation instructions should be explained in the README file
+#   i.e. which selections to use to generate the link. 
+
+#Link gives all totals excluding regional data 
+total_link <- "https://www.nomisweb.co.uk/api/v01/dataset/NM_161_1.data.csv?geography=2092957715,2092957699,2092957700,2092957703,2092957716&cause_of_death=9700,560,9,1130,1140,1150,1160,1170,1180,11095,11096,11098&gender=0&age=0...20&measure=2&measures=20100"
+
```

Comment:
I think this link is missing covid (LC47) - is it meant to?
Because Regions doesn't fit in the 25000 row limit, we would need 3 new links (region total, then by male and female)

I think a better approach is to add e.g. `years < - c(2020, 2021)` as a configuration, then run the code once for each year (loop through the years list). Then just have two links e.g. `all_but_female_data` and `female_data` and don't specify date in the link (which is what happens when you select all years in the nomis selection anyway). "&date=", year is then added to the end of the link when looping through them (see comment in compile_tables)

Reply:
Yes, makes sense and have done. 

Covid has recently been added to the UN metadata. Now included.